liblepton/scheme/Makefile.am

```diff
@@ -1,11 +1,22 @@
 
 scmdatadir = $(GEDADATADIR)/scheme
 nobase_dist_scmdata_DATA = \
-	geda.scm geda-deprecated-config.scm color-map.scm \
-	geda/object.scm geda/page.scm geda/attrib.scm geda/deprecated.scm \
-	geda/os.scm geda/config.scm geda/log-rotate.scm geda/log.scm \
-	geda/library.scm geda/repl.scm \
-	lepton/version.scm
+	geda.scm \
+	geda-deprecated-config.scm \
+	color-map.scm \
+	geda/object.scm \
+	geda/page.scm \
+	geda/attrib.scm \
+	geda/deprecated.scm \
+	geda/os.scm \
+	geda/config.scm \
+	geda/log-rotate.scm \
+	geda/log.scm \
+	geda/library.scm \
+	geda/repl.scm \
+	lepton/version.scm \
+	conf/keys.scm \
```

Comment:
I would suggest to rename `conf/keys.scm` to `lepton/legacy-config/keylist.scm`, and `conf/upgrade.scm` to `lepton/legacy-config.scm`, so that the modules were in the `lepton` namespace, and longer, more descriptive names were used for files and procedures.